.github/workflows/integration-tests-win.yml

```diff
@@ -0,0 +1,21 @@
+name: Integration Tests Windows
+on:
+  # Run tests on main just so the module and build cache can be saved and used
+  # in PRs. This speeds up the time it takes to test PRs dramatically.
+  # (More information on https://docs.github.com/en/enterprise-server@3.6/actions/using-workflows/caching-dependencies-to-speed-up-workflows)
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  run_tests:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Setup Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: "1.22"
```

Comment:
Would that become another place where we need to update Go version? What's the process currently to update these?

Reply:
yes, and afaik it's just "search and replace" through the codebase :/